src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS
+            = {"red", "green", "blue", "yellow", "pink", "black", "white"};
+
+    public String getRandomColor() {
+        Random random = new Random();
```

Comment:
Consider moving the instantiation of 'Random' to a class-level field. This way, you won't need to create a new 'Random' object every time you call 'getRandomColor()', which can be more efficient.